### ISSUE TITLE
Parse error: http://www.hyperorg.com/blogger/2012/12/18/misc-i-bet-your-ontology-never-thought-of-this-one/

### ISSUE THREAD

**Issue by gimsieke** [ftrain/savepublishing#58]
Several full stops unrecognized & layout gets messed up

On second viewing, it turns out that the layout clutter might be related to the banner that appears dynamically when you’ve reached the bottom of the page.

And there was actually one full stop that wasn’t considered as such, and it was after a single letter B which could have been regarded as an abbreviation. 

So I think you  can close this one (unless you feel the urge to do something about it, of course).

**Comment by ftrain** [comment]
Nah, let's leave it open for further study. Even on pages with crazy DOM stuff going on this bookmarklet shouldn't cause that much interference.
